sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -287,10 +284,7 @@ Analyze text, selection marks, table structures, styles, paragraphs, and key-val
 ```C# Snippet:FormRecognizerAnalyzePrebuiltDocumentFromUriAsync
 Uri fileUri = new Uri("<fileUri>");
 
-AnalyzeDocumentOperation operation = await client.StartAnalyzeDocumentFromUriAsync("prebuilt-document", fileUri);
-
-await operation.WaitForCompletionAsync();
-
+AnalyzeDocumentOperation operation = await client.AnalyzeDocumentFromUriAsync(WaitUntil.Completed, "prebuilt-document", fileUri);
```

Comment:
>AnalyzeDocumentFromUriAsync [](http://example.com/codeflow?start=50&length=27)

All our code snippets use the WaitUntil.Complete pattern. Should we add code snippets or perhaps a sample for the other option here?